doc/whats_new/v0.20.rst

```diff
@@ -224,6 +224,15 @@ Linear, kernelized and related models
   underlying implementation is not random.
   :issue:`9497` by :user:`Albert Thomas <albertcthomas>`.
 
+Decomposition, manifold learning and clustering
+
+- Deprecate ``precomputed`` parameter in function
+  :func:`manifold.t_sne.trustworthiness`. Instead, the new parameter
```

Comment:
If we have a separate entry for enhancement it should be written as such. For example, trustworthiness now accepts a metric other than Euclidean.